src/sb_vision/src/ipm_filter.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: June 15, 2017
+ * Description: Filters out all non-green from the image
+ *              then publishes the new filtered image.
+ */
+
+#include "../include/IPMFilterNode.h"
```

Comment:
nitpicky but we can avoid bugs

Reply:
Not nitpicky at all! This is exactly the sort of thing these reviews are supposed to catch.